containeranalysis/google/cloud/devtools/containeranalysis_v1/gapic/container_analysis_client.py

```diff
@@ -39,6 +39,9 @@
 from google.iam.v1 import policy_pb2
 
 
+from grafeas import grafeas_v1
+from grafeas.grafeas_v1.gapic.transports import grafeas_grpc_transport
+
```

Comment:
If `grafeas` is *not* supposed to be an unconditional dependency, then these two imports need to be wrapped:

```python
try:
    from grafeas import grafeas_v1
    from grafeas.grafeas_v1.gapic.transports import grafeas_grpc_transport
except ImportError:
    grafeas_v1 = None
    grafeas_grpc_transport = None
```